gum/backend-darwin/gumprocess-darwin.c

```diff
@@ -192,6 +200,12 @@ static void gum_cpu_context_to_darwin (const GumCpuContext * ctx,
     gum_thread_state_t * state);
 static const char * gum_symbol_name_from_darwin (const char * s);
 
+static void
+gum_process_enumerate_malloc_ranges_helper (task_t task,
```

Comment:
Function prototypes should not have a newline after their return type, it's only the definitions that follow that convention.